cmd/go-runner/plugins/say.go

```diff
@@ -65,6 +65,16 @@ func (p *Say) RequestFilter(conf interface{}, w http.ResponseWriter, r pkgHTTP.R
 	}
 }
 
-func (p *Say) ResponseFilter(interface{}, pkgHTTP.Response) {
+func (p *Say) ResponseFilter(conf interface{}, w pkgHTTP.Response) {
```

Comment:
We need to use a new plugin to test the Response filter, as the route with Say plugin will not run the ext-plugin-post-resp plugin. It will be intercepted before the request reaches the upstream.